packages/utils/src/object/findKey/index.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ *
+ * @description 객체에서 조건에 부합하는 key를 반환합니다.
+ *
+ * @param obj - 검색하고자 하는 객체입니다.
+ * @param condition - 검색하고자 하는 조건입니다.
+ * @returns {string | undefined} - 검색하고자 하는 조건에 부합하는 key를 반환합니다. 만약 조건에 부합하는 key가 없다면 undefined를 반환합니다.
+ *
+ * @example
+ *
+ * const obj = {
+ *   bike: { active: true },
+ *   plane: { active: true },
+ *   car: { active: false },
+ * };
+ *
+ *  findKey(obj, (item) => item.active); // 'bike'
+ *  findKey(obj, (item) => item.inactive); // undefined
+ */
```

Comment:
```suggestion
/**
 * @description 객체에서 조건에 부합하는 key를 반환합니다.
 *
 * @param obj - 검색하고자 하는 객체입니다.
 * @param condition - 검색하고자 하는 조건입니다.
 * @returns {string | undefined} - 검색하고자 하는 조건에 부합하는 key를 반환합니다. 만약 조건에 부합하는 key가 없다면 undefined를 반환합니다.
 *
 * @example
 *
 * const obj = {
 *   bike: { active: true },
 *   plane: { active: true },
 *   car: { active: false },
 * };
 *
 *  findKey(obj, (item) => item.active); // 'bike'
 */
```
`findKey(obj, (item) => item.inactive); // undefined` 예제는 실제로 타스에서 타입 에러가 발생하기 때문에 예제에서 제외합니다.